gnuradio-runtime/lib/pmt/pmt_serialize.cc

```diff
@@ -744,7 +746,11 @@ pmt_t parse_pair(std::streambuf& sb)
     while (1) {
         expr = deserialize(sb); // read the car
 
-        nptr = cons(expr, PMT_NIL); // build new cell
+        if (type == PST_DICT)
```

Comment:
```suggestion
        if (type == PST_DICT) {
```

yeah, we have places without braces, but we really don't like them and @marcusmueller has been trying to fight them with clang-tidy, so please don't introduce new ones.

Reply:
good to know! i'll be sure to keep this in mind (and learn how to use clang-tidy at some point...); thanks